Error while compiling for web.

I get an error while compiling for web, i have installed `emsdk` from AUR, but i get the following error:

`make: emcc: No such file or directory.`
`make: *** [Makefile:735: build/us_web/src/engine/behavior_script.o] Error 127`

 
- OS: Arch Linux 64-bit

--- a/include/libc/stdlib.h
+++ b/include/libc/stdlib.h
@@ -1,6 +1,8 @@
 #ifndef STDLIB_H
 #define STDLIB_H
 
+#ifndef TARGET_WEB
+
 typedef struct lldiv_t
 {
     long long quot;
@@ -15,5 +17,11 @@ typedef struct ldiv_t
 
 lldiv_t lldiv(long long num, long long denom);
 ldiv_t ldiv(long num, long denom);
+
+#else
+
+#include <stdlib.h>
+
+#endif
 
 #endif

--- a/src/engine/math_util.h
+++ b/src/engine/math_util.h
@@ -29,7 +29,11 @@ extern f32 gCosineTable[];
 
 #define sqr(x) ((x) * (x))
 
+#ifndef TARGET_WEB
 #define abs(x) ((x) < 0 ? -(x) : (x))
+#else
+#include "../../include/libc/stdlib.h"
+#endif
 
 void *vec3f_copy(Vec3f dest, Vec3f src);
 void *vec3f_set(Vec3f dest, f32 x, f32 y, f32 z);

--- a/src/game/mario_actions_cutscene.c
+++ b/src/game/mario_actions_cutscene.c
@@ -3,7 +3,6 @@
 #include "prevent_bss_reordering.h"
 #include "sm64.h"
 #include "gfx_dimensions.h"
-//#include "game.h"
 #include "game_init.h"
 #include "sound_init.h"
 #include "level_update.h"

--- a/src/game/options_menu.c
+++ b/src/game/options_menu.c
@@ -23,6 +23,7 @@
 #include "pc/controller/controller_api.h"
 
 #include <stdbool.h>
+#include "../../include/libc/stdlib.h"
 
 u8 optmenu_open = 0;
 

